CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+# 0.10.0
+
+* Add support for running on a host with cgroup v2 (unified cgroup hierarchy) enabled. ([#76])
+* Fix systemd warning about conflicting jobs when the admin container exits. ([#79])
+* Pick up latest AL2 updates.
```

Comment:
This usually goes without saying?

Reply:
The change log is mixed on this. If it's the only change for a new container image, it's been called out so far. If other changes happened and updates were picked up anyway, it's not been called out.